operational/vmware/instance_tag_sync/changelog.md

```diff
@@ -1,8 +1,14 @@
-v1.1
-----
+# Changelog
+
+## v1.3
+
+- remove permissions block and adding incident resource table
```

Comment:
There were no changes to permissions block.

Reply:
made appropriate changes